Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/requirements.txt

```diff
@@ -7,4 +7,4 @@ transformers
 sklearn
```

Comment:
I think the transformers pin is needed
```transformers==3.5.0```
Not sure if this breaks deployment again or not

Reply:
see issue #13 if it does break deployment

Reply:
Logwise it would be the same, since `torchserve` writes to stdout in both cases.  But the `fg` solution might be cleaner in a sense that if `torchserve` dies the container would die as well (and that's what we want). I'll update the code here.